### PR TITLE
Kafka-queue

### PR DESCRIPTION
**Pull request by huymnguyen24**

### CODE REVIEW

**Comment by huymnguyen24** [comment]
To run the test please bring up to docker-compose using `docker compose up` command

**Review by DedsecKnight — APPROVED** [review]
lgtm! nicely done :+1:

File: kafka-queue/kafka_util.go [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+	"github.com/confluentinc/confluent-kafka-go/kafka"
+)
+
+func NewKafkaProducer(kafkaServer string) *kafka.Producer {
+	producer, err := kafka.NewProducer(&kafka.ConfigMap{
+		"bootstrap.servers": kafkaServer,
+		"acks":              "all"})
+	if err != nil {
+		fmt.Println("Failed to create producer due to ", err)
+		panic(err)
```
Comment (1solomonwakhungu) [inline kafka-queue/kafka_util.go]: Panicking in code will stop the program. 
In production, this service will stop working is there is an error.
Best to print the error and return.

File: kafka-queue/kafka_util.go [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+	case *kafka.Error:
+		return errors.New("Failed to produce message due to " + event.Error())
+	}
+	return errors.New("Failed to produce message due to unknown error")
+}
+
+func NewKafkaConsumer(kafkaServer string, kafkaTopic string, workerID string) *kafka.Consumer {
+	consumer, initConsumerErr := kafka.NewConsumer(&kafka.ConfigMap{"bootstrap.servers": kafkaServer, "group.id": workerID, "auto.offset.reset": "smallest"})
+	if initConsumerErr != nil {
+		panic(errors.New("Unable to create kafka consumer due to error " + initConsumerErr.Error()))
```
Comment (1solomonwakhungu) [inline kafka-queue/kafka_util.go]: Another panic here.

**Review by 1solomonwakhungu — CHANGES_REQUESTED** [review]

**Review by 1solomonwakhungu — APPROVED** [review]
👍

**Pull request merged**